packages/react/src/components/SuiteHeader/__snapshots__/SuiteHeader.story.storyshot

```diff
@@ -4294,6 +4294,45 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/S
       <div
         className="bx--header__global"
       >
+        <button
```

Comment:
There's a snapshot failure after the auto-overflow menu PR was merged. We'll also want to check that the overflow functionality still works as expected because it's built to check for overlap between the first icon in the action items and the div containing the title/subtitle. I don't know how it'll work with this new hidden button in the first position.

Reply:
Yeah, we'll need to account for the icon in some way. The deploy preview shows the [loading state story](https://deploy-preview-2335--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-suiteheader--loading-state) always having overflow icon and the [next story](https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-suiteheader--loading-state) works as expected and the overflow only appears at smaller screen sizes.

Reply:
Fixed by replacing `display: none` with `visibility: hidden` in the first actionItem in SuiteHeader.